server/trafficrules/pom.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.3.3.RELEASE</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>com.coffeejug</groupId>
+	<artifactId>trafficrules</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>trafficrules</name>
+	<description>Traffic rules game for children</description>
+
+	<properties>
+		<java.version>1.8</java.version>
```

Comment:
What about using the latest Java version? Java 1.8 is pretty old ;)

Reply:
What for ? 1.8 is quite enough for our tasks...

Reply:
Event 1.6 is quite enough for our task or even PHP, but why using the 7-year old version when we can use the brand new one?